src/utils/hostConfig.js

```diff
@@ -32,7 +32,7 @@ export default {
   },
 
   appendInitialChild(parentInstance: GenericParentInstance, child: any) {
-    parentInstance.appendChild(child);
+    parentInstance.appendChild(child, true);
```

Comment:
how about having `parentInstance.appendInitialChild(child);` instead of making the check with every fn call?